spec/tokenized-buffer-spec.coffee

```diff
@@ -580,7 +572,7 @@ describe "TokenizedBuffer", ->
 
     describe "when the selector matches a run of multiple tokens at the position", ->
       it "returns the range covered by all contigous tokens (within a single line)", ->
-        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.function', [1, 18])).toEqual [[1, 6], [1, 28]]
+        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.meta.function', [1, 18])).toEqual [[1, 6], [1, 28]]
```

Comment:
I think this is destructive change on selector if this doesn't accept `.function` selector anymore and we should use `.meta.function` instead.

This caused bug on the plugin because it uses `.regexp` instead of `.string.regexp`.
https://github.com/klorenz/atom-regex-railroad-diagrams/issues/42